adding sakura workshop fuji75 hotswap and solder pcb supports

CMMS-Freather opened the pull request:
<!--- Provide a general summary of your changes in the title above. -->
adding sakura workshop fuji75 hotswap and solder pcb supports
<!--- This template is entirely optional and can be removed, but is here to help both you and us. -->
<!--- Anything on lines wrapped in comments like these will not show up in the final text. -->

adding sakura workshop fuji75 hotswap and solder pcb supports

<!--- Describe your changes in detail here. -->

## QMK Pull Request 
https://github.com/qmk/qmk_firmware/pull/22158
<!--- VIA support for new keyboards MUST be in QMK master already -->

<!--- Add link to QMK Pull Request here. -->

<!--- THIS IS MANDATORY. -->

<!--- IF THERE IS NO LINK TO SHOW VIA SUPPORT IS IN QMK MASTER ALREADY, -->
<!--- THIS PR WILL BE CLOSED IMMEDIATELY FOR WORKFLOW REASONS.  -->

## Checklist

<!--- Put an `x` in all the boxes that apply. -->

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

Xelus22 reviewed (changes requested):
Everything looks good. 

Just the `Fuji75` folder is uppercase. Please change to lowercase to reflect QMK tree structure/naming

CMMS-Freather commented:
> Everything looks good.
> 
> Just the `Fuji75` folder is uppercase. Please change to lowercase to reflect QMK tree structure/naming

renamed and updated
